java/core/src/main/java/com/whylogs/core/schemas/ColumnSchema.java

```diff
@@ -0,0 +1,24 @@
+package com.whylogs.core.schemas;
+
+import com.whylogs.core.metrics.Metric;
+import com.whylogs.core.metrics.MetricConfig;
+import com.whylogs.core.resolvers.Resolver;
+import java.lang.reflect.Type;
+import java.util.HashMap;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
```

Comment:
I take it the `@Data` decorator makes the getters for the private members available?

Reply:
Yup! :)